app/controllers/questions_controller.rb

```diff
@@ -0,0 +1,49 @@
+class QuestionsController < ApplicationController
+  before_action :set_question, only: [:show, :update, :destroy]
+
+  def index
+    questions = Question.all
+    render json: questions
+  end
+
+  def show
+      render json: @question, status: 200
+  end
+
+  def create
+    question = Question.new(questions_params)
+    if question.save
+      render json: question, status: :created
+    else
+      render json:  question.errors, status: 400
+    end
+  end
+
+  def update
+    if @question.update(questions_params)
+      render json: @question, status: :ok
+    end
+  end
```

Comment:
Just wondering; what happens _if @question.update()_ is not successful? :thought_balloon: